addons/account/models/ir_actions_report.py

```diff
@@ -17,7 +17,7 @@ def _render_qweb_pdf_prepare_streams(self, report_ref, data, res_ids=None):
             return super()._render_qweb_pdf_prepare_streams(report_ref, data, res_ids=res_ids)
 
         invoices = self.env['account.move'].browse(res_ids)
-        if any(x.move_type not in ('in_invoice', 'in_receipt') for x in invoices):
+        if any(x.move_type not in ('in_invoice', 'in_refund') for x in invoices):
```

Comment:
Why changing `in_receipt` for `in_refund`?
You can add it so :)
Please use the helper directly (`is_purchase_document(include_receipts=True)`)